canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx

```diff
@@ -168,6 +174,20 @@ class PropertiesMain extends React.Component {
 		return PropertyUtils.formatMessage(this.props.intl, MESSAGE_KEYS.PROPERTIESEDIT_REJECTBUTTON_LABEL);
 	}
 
+	getEditorSize() {
+		// Determine whether to persist initialEditorSize or set the defaultEditorSize in certain cases
+		const defaultEditorSize = this.propertiesController.getForm().editorSize;
```

Comment:
Do you know how this works if someone uses the pixel size instead of small, medium, large?

Reply:
If `editor_size` is not specified in the paramDef.json, default `edior_size` is set to `small`. And rightFlyoutPanel has width set to `pixel_width.min`.


For example -
`Filter_paramDef.json` doesn't have `editor_size`. But it has `pixel_width: { "min": 200, "max": 700 }`.
If persisted editor_size is `small`, right flyout panel has `width: 200px`
```
<aside aria-label="Properties" role="complementary" class="properties-wrapper properties-right-flyout properties-small" tabindex="0" style="width: 200px;">
```
If persisted editor_size is `medium`, right flyout panel has `width: 700px`
```
<aside aria-label="Properties" role="complementary" class="properties-wrapper properties-right-flyout properties-medium" tabindex="0" style="width: 700px;">
```
If persisted editor_size is `large`, default editor_size is set to `small` and right flyout panel has `width: 200px`
```
<aside aria-label="Properties" role="complementary" class="properties-wrapper properties-right-flyout properties-small" tabindex="0" style="width: 200px;">
```